### PR TITLE
docs(readme): Update installation instruction

### PR DESCRIPTION
**Pull request by chawyehsu**
#### Description

Update installation instruction to reflect adoption of the standalone installer.

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,60 +56,15 @@ scoop install python ruby go perl
...
 ## Installation
 
-Run the following command from your PowerShell to install scoop to its default location (`C:\Users\<user>\scoop`)
+Run the following command from a **non-admin** PowerShell to install scoop to its default location `C:\Users\<YOUR USERNAME>\scoop`.
 
 ```powershell
-Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')
-
-# or shorter
 iwr -useb get.scoop.sh | iex
```
Comment (rashil2000) [inline README.md:64]:
```suggestion
iwr get.scoop.sh | iex
```

I don't think this param is required anymore

File: README.md [hunk 2]
```diff
@@ -56,60 +56,15 @@ scoop install python ruby go perl
...
 ```
 
-Once installed, run `scoop help` for instructions.
-
-The default setup is configured so all user installed programs and Scoop itself live in `C:\Users\<user>\scoop`.
-Globally installed programs (`--global`) live in `C:\ProgramData\scoop`.
```
Comment (rashil2000) [inline README.md:79]: These lines should be kept

**Comment by rashil2000** [comment]
Any updates on https://github.com/ScoopInstaller/Scoop/issues/4783#issuecomment-1065127304?

File: README.md (hunk 1 above)
Comment (chawyehsu) [reply README.md:64]: It's required by PowerShell 5.

File: README.md (hunk 2 above)
Comment (chawyehsu) [reply README.md:79]: Once installed, the help message `Type 'scoop help' for instructions.` will be printed. And we've had a link to [documentation](https://github.com/ScoopInstaller/Scoop/pull/4825/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69) in the following section.

**Comment by chawyehsu** [comment]
> Any updates on [#4783 (comment)](https://github.com/ScoopInstaller/Scoop/issues/4783#issuecomment-1065127304)?

Use the `-RunAsAdmin` switch. Maybe we should maintain an official actions for setting up Scoop in GitHub Actions.

File: README.md [hunk 3]
```diff
@@ -56,60 +56,15 @@ scoop install python ruby go perl
 
 If you've built software that you'd like others to use, Scoop is an alternative to building an installer (e.g. MSI or InnoSetup) — you just need to zip your program and provide a JSON manifest that describes how to install it.
 
-## Requirements
-
-- Windows 7 SP1+ / Windows Server 2008+
-- [PowerShell 5](https://aka.ms/wmf5download) (or later, include [PowerShell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6)) and [.NET Framework 4.5](https://www.microsoft.com/net/download) (or later)
-- PowerShell must be enabled for your user account e.g. `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
-
```
Comment (niheaven) [inline README.md:64]: Why removes these requirements?

BTW, the PowerShell version should be written as:

```markdown
- [Windows PowerShell 5.1](https://aka.ms/wmf5download) (or [PowerShell](https://aka.ms/powershell))
- [.NET Framework 4.5](https://www.microsoft.com/net/download) or later
```

Comment (chawyehsu) [reply README.md:64]: The standalone installer's readme stated these requirements. I'm trying to simplify the instruction here so we don't have to maintain different installation instruction copies. Users should be able to find the complete instruction via [the link here](https://github.com/ScoopInstaller/Scoop/pull/4825/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R67). I would update the requirements per your suggestion to https://github.com/ScoopInstaller/Install#prerequisites and the website.

**Review by niheaven — APPROVED** [review]

**Review by niheaven — APPROVED** [review]

**Pull request merged**